<launch>
<group ns = "leader_ukf">
   <node pkg="UKF" name="ukf" type="ukf_estimate" output="screen" >
     <param name="topic_imu" value="/drone3/mavros/imu/data" />
     <param name="topic_mocap" value="/vrpn_client_node/RigidBody3/pose" />
     <param name="topic_thrust" value="/drone3/mavros/vfr_hud" />
     <param name="imu_bias_x" value="-0.060972" />
     <param name="imu_bias_y" value="0.024664" />
     <param name="imu_bias_z" value="-0.042591" />
     <param name="g" value="9.80665" />
     <param name="thrust" value="0.430" />
     <param name="m" value="1.25" />
   </node>
</group>
<group ns = "follower_ukf">
   <node pkg="UKF" name="ukf" type="ukf_estimate" output="screen" >
     <param name="topic_imu" value="/drone2/mavros/imu/data" />
     <param name="topic_mocap" value="/vrpn_client_node/RigidBody2/pose" />
     <param name="topic_thrust" value="/drone2/mavros/vfr_hud" />
     <param name="imu_bias_x" value="-0.022596" />
     <param name="imu_bias_y" value="-0.041609" />
     <param name="imu_bias_z" value="-0.090096" />
     <param name="g" value="9.80665" />
     <param name="thrust" value="0.460" />
     <param name="m" value="1.25" />
   </node>
</group>
</launch>
